test/OpenTelemetry.Instrumentation.EntityFrameworkCore.Tests/DatabaseSemanticConventionHelperTests.cs

```diff
@@ -0,0 +1,79 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+using Microsoft.Extensions.Configuration;
+using Xunit;
+using static OpenTelemetry.Internal.DatabaseSemanticConventionHelper;
+
+namespace OpenTelemetry.Internal.Tests;
+
+public class DatabaseSemanticConventionHelperTests
```

Comment:
These tests should be moved to the OpenTelemetry.Tests project. Similar to how we had it with the HTTP conventions https://github.com/open-telemetry/opentelemetry-dotnet/pull/5270/files#diff-ece803150e7b62366d8f76174609124fad529370a8c9a62021644dfe43837c45L10